packages/cli/src/lib/setup/setupList.ts

```diff
@@ -0,0 +1,916 @@
+/**
+ *      List different objects for CLI
+ *
+ *      Copyright 2013-2022 bluefox <dogafox@gmail.com>
+ *
+ *      MIT License
+ *
+ */
+
+import { EXIT_CODES } from '@iobroker/js-controller-common';
+import * as fs from 'fs-extra';
+import { tools } from '@iobroker/js-controller-common';
+import type { Client as StatesRedisClient } from '@iobroker/db-states-redis';
+import type { Client as ObjectsRedisClient } from '@iobroker/db-objects-redis';
+
+interface File {
+    adapter: string;
+    path: string;
+    file: Record<string, any>;
+}
+
+interface IdValueObject {
+    id: string;
+    value: any;
+}
+
+interface FlagObject {
+    alive?: boolean;
+    enabled?: boolean;
+    disabled?: boolean;
+    port?: number;
+    ssl?: boolean;
+    ip?: string;
+}
+
+export class List {
+    private config: Record<string, any>;
```

Comment:
We could think about adding a proper interface which describes the shape of the system config. Probably not part of this PR though